ArduCopter/Log.cpp

```diff
@@ -310,7 +310,7 @@ void Copter::Log_Write_Heli()
         LOG_PACKET_HEADER_INIT(LOG_HELI_MSG),
         time_us                 : AP_HAL::micros64(),
         desired_rotor_speed     : motors->get_desired_rotor_speed(),
-        main_rotor_speed        : motors->get_main_rotor_speed(),
```

Comment:
Perhaps `get_main_rotor_speed()` should be calling rpm?  Seems a useful abstraction.

Reply:
so we did not have a way of getting rotor speed to our motors class.  So  rotor speed was contrived based on a timer in motors.  Not many users probably use a hall effects sensor to get rpm and the timer was a substitution for this.   I hope to change all of this once we get a reliable method to determine rotor speed without the user having to put a hall effects sensor on the aircraft.   So this will be changed once we get the FFT working.    this commit will be removed before merging.